File: src/main/java/de/storchp/opentracks/osmplugin/utils/TrackPointsDebug.java
package de.storchp.opentracks.osmplugin.utils;

public class TrackPointsDebug {
    public int trackpointsReceived = 0;
    public int trackpointsInvalid = 0;
    public int trackpointsDrawn = 0;
    public int trackpointsPause = 0;
    public int segments = 0;

    public void add(final TrackPointsDebug other) {
        this.trackpointsReceived += other.trackpointsReceived;
        this.trackpointsInvalid += other.trackpointsInvalid;
        this.trackpointsDrawn += other.trackpointsDrawn;
        this.trackpointsPause += other.trackpointsPause;
        this.segments += other.segments;
    }
}

File: src/main/kotlin/de/storchp/opentracks/osmplugin/utils/TrackPointsDebug.kt
package de.storchp.opentracks.osmplugin.utils

class TrackPointsDebug {
    var trackpointsReceived: Int = 0
    var trackpointsInvalid: Int = 0
    var trackpointsDrawn: Int = 0
    var trackpointsPause: Int = 0
    var segments: Int = 0

    fun add(other: TrackPointsDebug) {
        this.trackpointsReceived += other.trackpointsReceived
        this.trackpointsInvalid += other.trackpointsInvalid
        this.trackpointsDrawn += other.trackpointsDrawn
        this.trackpointsPause += other.trackpointsPause
        this.segments += other.segments
    }
}
